### PR TITLE
🚣🏿‍♀️ Add link to resource form message

### PR DESCRIPTION
**Pull request by nkonjeti**
## Description
In success message after editing or creating resource the link to the resource is displayed and can be clicked.

Fixes #<number>

## Screenshots
Screen Shot 2021-03-25 at 8 59 34 PM

### CODE REVIEW

File: client/src/components/ResourceForm.jsx [hunk 1]
```diff
@@ -14,13 +14,23 @@ import '../css/ResourceForm.css';
 
 const { Content } = Layout;
 
+const linkResource = (resourceId, type) => (
+  <>
+    <div>
```
Comment (n3a9) [inline client/src/components/ResourceForm.jsx:19]: What's the purpose of this div?

File: client/src/components/ResourceForm.jsx [hunk 2]
```diff
@@ -14,13 +14,23 @@ import '../css/ResourceForm.css';
...
+      {' '}
```
Comment (n3a9) [inline client/src/components/ResourceForm.jsx:20]: Is this what's causing the new line?

**Review by n3a9 — COMMENTED** [review]
Great job Neha! Overall I think that we can make the message shorter so that the check mark and message are in the same box. Maybe something like "Resource created! Click here to view."

**Review by jackieo5023 — APPROVED** [review]
Yayy!!!!

**Review by n3a9 — APPROVED** [review]
So beautiful :)

**Pull request merged**